src/org/usfirst/frc/team2585/robot/Commands.java

```diff
@@ -59,31 +59,62 @@ private static void stop() {
 	
 	
 	/**
-	 * Autonomous command that that drives the robot forward and then turns left and continues driving forward
+	 * Autonomous command that drives according to its position and the side of its switch
 	 */
 	public class Main implements AutonomousCommand {
+		private static final int timeToDriveStraight = 3100;
+		
+		private void runFromMiddleToLeft(long timeElapsed) {
```

Comment:
There should instead be one runFromMiddle() and one runFromSide() method. Then inside those methods, we can check whether we are going left or right using the location and gameSpecificMessage.

This will make the code simpler because the if else statement will be reduced, and we will have to write fewer methods. When we eventually make statements that involve the timing of driving forward, turning inwards, and depositing, it will be easier if we can say
if (left) turn left
else turn right
Rather than having two separate methods